metadata-ingestion/src/datahub/cli/quickstart_versioning.py

```diff
@@ -58,20 +46,36 @@ def _fetch_latest_version(cls) -> str:
         return json.loads(response.text)["tag_name"]
 
     @classmethod
-    def fetch_quickstart_config(cls):
-        response = None
+    def fetch_quickstart_config(cls) -> "QuickstartVersionMappingConfig":
         config_raw = None
         try:
             response = requests.get(DEFAULT_REMOTE_CONFIG_PATH, timeout=5)
+            response.raise_for_status()
             config_raw = yaml.safe_load(response.text)
         except Exception:
-            click.echo("Couldn't connect to github")
-            path = os.path.expanduser(DEFAULT_LOCAL_CONFIG_PATH)
-            with open(path, "r") as f:
-                config_raw = yaml.safe_load(f)
+            logger.debug("Couldn't connect to github, trying to read from local file.")
```

Comment:
maybe it worth to log the exception as well

Reply:
updated